lib/include/public/LogManagerBase.hpp

```diff
@@ -731,7 +731,7 @@ namespace MAT_NS_BEGIN
         static ILogManager* GetInstance() noexcept
```

Comment:
This probably needs to go as well

Reply:
Really all the static methods

Reply:
Yes, I am trying to see if we can make LogManager as instance of LogManagerBase, and then make all these methods as non-static. Though LogConfiguration is still a concern here, as it is difficult to create deep copy of it (as it can contain `void *` as the config value).